internal/flags/apiflags/flags.go

```diff
@@ -21,22 +21,19 @@ import (
 	"github.com/spf13/pflag"
 
 	gencrd "github.com/operator-framework/operator-sdk/internal/generate/crd"
-	"github.com/operator-framework/operator-sdk/internal/util/projutil"
 )
 
 type APIFlags struct {
-	SkipGeneration   bool
-	APIVersion       string
-	Kind             string
-	CrdVersion       string
-	HelmChartRef     string
-	HelmChartVersion string
-	HelmChartRepo    string
+	SkipGeneration bool
+	APIVersion     string
+	Kind           string
+	CrdVersion     string
 }
 
-// AddTo - Add the reconcile period and watches file flags to the the flagset
+// AddToNew - Add the reconcile period and watches file flags to the the flagset
```

Comment:
Do we need `AddToNew`? It looks the same as `AddTo` now that the Helm fields have been removed.